app/views/content/train-to-be-a-teacher/_questions.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<section class="category__questions">
+    <div class="category__question">
+        <h2>
+            <strong>What subject</strong> do you want to teach?
+        </h2>
+
+        <p>
+            If you want to teach a particular subject there may be extra
+            training you can do.
+        </p>
+
+        <a href="#" class="button">Explore subject enhancement training</a>
+    </div>
+
+    <div class="category__question">
+        <h2>
+            <strong>Who</strong> do you want to teach?
+        </h2>
+
+        <p>
+            If you're not sure what age or student group you want to
+            teach we can help you work it out and find the right training.
+        </p>
+
+        <a href="#" class="button">Find out about training</a>
```

Comment:
I know there isn't a page for this yet, but the link needs to be more specific when there is :-)